dist/stat_data.py

```diff
@@ -334,6 +334,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_server_skip_dirty_pages_during_checkpoint', 'eviction server skips dirty pages during a running checkpoint'),
     CacheStat('cache_eviction_server_skip_pages_retry', 'eviction server skips pages that previously failed eviction and likely will again'),
     CacheStat('cache_eviction_server_skip_pages_last_running', 'eviction server skips pages that are written with transactions greater than the last running'),
+    CacheStat('cache_eviction_server_skip_parent_intl_page_of_last_seen_page', 'eviction server skip parent internal page of last seen page.'),
```

Comment:
How about?
`eviction server skips internal pages as it has an active child.`

Reply:
Fixed in 1057715880393100056c1876e86de15d263961f8